astropy/time/formats.py

```diff
@@ -710,11 +711,12 @@ class TimeUnix(TimeFromEpoch):
 
 class TimeUnixTai(TimeUnix):
     """
-    Seconds from 1970-01-01 00:00:08 TAI (see notes), including leap seconds.
+    Unix time (TAI): SI seconds from 1970-01-01 00:00:08 TAI, increases monotonically.
 
     This will generally differ from Unix time by the cumulative integral number
-    of leap seconds since 1970-01-01 UTC.  This convention matches the definition
-    for linux CLOCK_TAI (https://www.cl.cam.ac.uk/~mgk25/posix-clocks.html).
+    of leap seconds introduced into UTC since 1970-01-01 UTC.
```

Comment:
Please maintain word wrapping at 80 columns for all docstring modifications. These docstrings are rendered verbatim when asking for help in IPython.